src/index.js

```diff
@@ -4,11 +4,12 @@ var Debugger = require('./ui/Ethdebugger')
 var BasicPanel = require('./ui/BasicPanel')
 var TraceManager = require('./trace/traceManager')
 var CodeManager = require('./code/codeManager')
+var SourceMappingDecoder = require('./util/sourceMappingDecoder')
 
 if (typeof (module) !== 'undefined' && typeof (module.exports) !== 'undefined') {
   module.exports = modules()
 }
-if (window) {
+if (typeof (window) !== 'undefined') {
   window.remix = modules()
```

Comment:
I think we should not support the second use-case. A package loader has to handle that.